modules/victoria-metrics/templates/values.yaml

```diff
@@ -16,7 +16,7 @@ victoria-metrics-operator:
   cleanupImage:
     repository: bitnami/kubectl
     # use image tag that matches k8s API version by default
-    # tag: 1.29.6
+    tag: 1.30.2
```

Comment:
Is pinning this version better than using the default as was done before?

Reply:
There was a bug that I had reported around this. Basically a '+' sign that was getting included in the tag image that prevented the cleanup job from running. I updated the helm chart to the newer version so we don't need to set this tag manually anymore. I will remove